Libraries/PeriphDrivers/Include/MAX32655/pt.h

```diff
@@ -169,39 +169,116 @@ uint32_t MXC_PT_IsActive(uint32_t pts);
 void MXC_PT_SetPattern(unsigned pts, uint32_t pattern);
 
 /**
- * @brief      Enable interrupts for the pulse trains selected.
+ * @brief      Enable Stop interrupts for the pulse trains selected.
+ * 
+ * @note       This is just an explicit version of MXC_PT_EnableInt()
  *
  * @param      pts   Bit mask of which pulse trains to enable. Set the bit
  *                   position of each pulse train to enable it. Bit0-\>pt0,
  *                   Bit1-\>pt1... etc, 1 will enable the interrupt, 0 to leave
  *                   a PT channel in its current state.
  */
-void MXC_PT_EnableInt(uint32_t pts);
+void MXC_PT_EnableStopInt(uint32_t pts);
```

Comment:
ME17 `Pulse_Train` example depends on `MXC_PT_EnableInt`.  Update example to use new functions to clear build warning